fv3gfs/physics/stencils/physics.py

```diff
@@ -56,6 +59,106 @@ def atmos_phys_driver_statein(
     with computation(PARALLEL), interval(0, 1):
         prsi = ptop
 
+    with computation(FORWARD), interval(1, None):
+        prsi = prsi[0, 0, -1] + delp[0, 0, -1]
+
+    with computation(PARALLEL), interval(0, -1):
+        prsik = log(prsi)
+        qvapor = qvapor / delp
```

Comment:
Not something to change in this PR, but I've been trying to refactor the code where possible to ensure that the units of a variable don't change. e.g. if the left has units of mixing ratio, then up until now qvapor does not have units of mixing ratio and is holding some other value. Something to consider when you're able to refactor and merge things later, without e.g. adding a new storage for each of these.